data/json/item_groups.json

```diff
@@ -6855,10 +6855,15 @@
     "id" : "helicopter",
     "magazine": 100,
     "items":[
+      { "group": "guns_launcher_milspec", "prob": 10 },
+      { "group": "guns_rifle_milspec", "prob": 10 },
+      { "item": "human_flesh", "prob": 4, "charges-min": 1, "charges-max": 10 },
+      { "item": "bone_human", "prob": 10, "charges-min": 1, "charges-max": 5 },
+      { "item": "rag_bloody", "prob": 10, "charges-min": 1, "charges-max": 40 },
```

Comment:
charges-min & charges-max mean something only for items with charges.
BTW, what ``` "magazine": 100,``` means for item group??!

Reply:
Magazine 100 means every gun always spawns with a magazine.